EssentialCSharp.Web/Views/Home/About.cshtml

```diff
@@ -37,7 +37,9 @@
     <div class="row gx-5">
         <div class="col-12 col-md-6">
             <div class="d-flex align-items-center justify-content-center h-100">
-                <img src="/images/book.png" class="img-fluid" style="height: 450px" alt="Essential C Sharp Book">
+                <a href="https://a.co/d/95wxhTg" target="_blank">
+                    <img src="/images/book.png" class="img-fluid" style="height: 450px" alt="Essential C Sharp Book">
+                </a>
```

Comment:
Looks like a rebase is needed or similar, these changes have already been merged (same with below file)

Reply:
Thanks for the heads up - it's fixed now.